docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -2005,6 +2005,7 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/security/oauth/smart/enabled`|boolean|Whether or not the server is enabled for OAuth-based authentication/authorization|
 |`fhirServer/security/oauth/smart/scopes`|array|The list of SMART scopes to advertise in the `.well-known/smart-configuration endpoint|
 |`fhirServer/security/oauth/smart/capabilities`|array|The list of SMART capabilities to advertise in the `.well-known/smart-configuration endpoint|
+|`fhirServer/security/operation/authorize/reindex`|boolean|True, the reindex operation is made available only to fhiradmins, and users are denied access|
```

Comment:
I understand how this could make sense under the `security` propertyGroup, but I think I'd prefer it to have its own `operations` securityGroup instead.  that would let us group it together with `fhirServer/core/disabledOperations` and would let us extend with additional properties in the future (if needed).

something like this:
```
    "fhirServer": {
        "operations": {
            "open": true,
            "reindex": {
                "enabled": false
            },
            "erase": {
                "enabled": true
                "rolesAllowed": ["FHIRUsers, FHIRAdmin"],
            }
        },
```

Reply:
done